pkg/graph/schema/types/mto_milestone.graphql

```diff
@@ -36,16 +36,12 @@ type MTOMilestone {
   addedFromMilestoneLibrary: Boolean! # Only true when commonMilestone && commonMilestone
   commonMilestone: MTOCommonMilestone
   solutions: [MTOSolution!]!
-  # Category: {
-  #   mtoCategoryID: UUID
-  #   subcategoryID: UUID
-  #   Category: MTOCategory
-  #   subCategory: MTOSubcategory
-  # }
-  # TODO (mto) do we need to resolve category? Should we need to resolve it here? Also, 
-  # category: UNION MTOCategory | MTOSubcategory ?
-  category: MTOCategory!
-  subCategory: MTOSubcategory!
+  """
+  Category resolves relational category information.
+  if a milestone is uncategorized, this struct will still return
+  it will have a null value for both
```

Comment:
Is this still accurate? My understanding of our code/approach is that it returns non-nullable types (i.e. the `Uncategorized` items) since the GQL is non-nullable for each of the fields in this type.

Reply:
good call, you are right! I forgot to update this